project/Dependencies.scala

```diff
@@ -21,20 +21,21 @@ object Dependencies {
     "com.getsentry.raven"            % "raven-logback"       % "7.8.6",
     "com.typesafe.scala-logging"    %% "scala-logging"       % "3.7.2",
 
-    "org.broadinstitute.dsde.vault" %% "vault-common"        % "0.1-19-ca8b927",
-    "org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-aab4976-SNAP"
+    "org.broadinstitute.dsde.vault" %% "vault-common"        % "0.1-19-ca8b927"
+      exclude("io.spray", "spray-routing_2.11"),
+    "org.broadinstitute.dsde"       %% "rawls-model"         % "0.1-9316d894-SNAP"
       exclude("com.typesafe.scala-logging", "scala-logging_2.11"),
     "org.broadinstitute.dsde.workbench" %% "workbench-util"  % "0.2-0b34b93",
 
-    "io.spray"                      %% "spray-can"           % sprayV,
-    "io.spray"                      %% "spray-routing"       % sprayV,
-    "io.spray"                      %% "spray-json"          % "1.3.3",
-    "io.spray"                      %% "spray-client"        % sprayV,
-    "io.spray"                      %% "spray-testkit"       % sprayV    % "test",
-    "com.typesafe.akka"             %% "akka-actor"          % akkaV,
-    "com.typesafe.akka"             %% "akka-testkit"        % akkaV     % "test",
-    "com.typesafe.akka"             %% "akka-slf4j"          % akkaV,
-    "com.typesafe.akka"             %% "akka-stream"         % akkaV,
+    "io.spray"                      %% "spray-can"                 % sprayV,
+    "io.spray"                      %% "spray-json"                % "1.3.3",
+    "io.spray"                      %% "spray-client"              % sprayV,
+    "io.spray"                      %% "spray-routing-shapeless23" % sprayV,
```

Comment:
What went down here was, the new Rawls model introduced libraries that rely on Shapeless 2.3.2, so I upgraded us from `spray-routing` to `spray-routing-shapeless23` to match.

At that point, `vault-common` still pulled in `spray-routing` and by extension Shapeless 1.2.4, so I just threw an exclusion on that.

Reply:
another option: we only use `ServerInitializer` and `ImplicitMagnet` from vault-common. We could pretty easily copy those into the orch codebase and remove the dependency on vault-common entirely. Definitely not required for this PR and probably more than you wanted to deal with - mentioning it for the sake of discussion.